httpserver/service_node.cpp

```diff
@@ -121,6 +120,32 @@ void ServiceNode::relay_batch(const std::string& data, sn_record_t sn) const {
         });
 }
 
+void ServiceNode::register_listener(const std::string& pk,
+                                    const std::shared_ptr<connection_t>& c) {
+    pk_to_listeners[pk].push_back(c);
+    BOOST_LOG_TRIVIAL(debug) << "register pubkey: " << pk
```

Comment:
Maybe we should move `obfuscate_pubkey` to common.h and commit to always using it for logs?

Reply:
And I guess potentially ifdef around for integration testing if you are using the full pk there?

Reply:
Maybe, but perhaps we could postpone it until we actually need it in other translation units?